.github/workflows/release.yaml

```diff
@@ -36,12 +36,20 @@ jobs:
 
       - name: Create a release candidate
         if: ${{ contains(github.ref, '-rc') }}
-        run: gh release create $TAG --prerelease --title $TAG --notes "Release candidate" ./out/ai-gateway-helm-${TAG}.tgz
```

Comment:
i think simply adding `./out/ai-gateway-crd-helm-${TAG}.tgz` to this line should be enough? the current change doesn't quite make sense to me.
```
USAGE
  gh release create [<tag>] [<files>...]
```